*** commons-daemon-1.0.13-src/src/native/unix/support/apsupport.m4.orig Mon Nov  4 20:43:08 2013
--- commons-daemon-1.0.13-src/src/native/unix/support/apsupport.m4      Mon Nov  4 20:42:57 2013
***************
*** 148,153 ****
--- 148,157 ----
      CFLAGS="$CFLAGS -DCPU=\\\"arm\\\""
      supported_os="arm"
      HOST_CPU=arm;;
+   aarch64*)
+     CFLAGS="$CFLAGS -DCPU=\\\"aarch64\\\""
+     supported_os="aarch64"
+     HOST_CPU=aarch64;;
    *)
      AC_MSG_RESULT([failed])
      AC_MSG_ERROR([Unsupported CPU architecture "$host_cpu"]);;
*** commons-daemon-1.0.13-src/src/native/unix/configure.orig    Mon Nov  4 20:40:54 2013
--- commons-daemon-1.0.13-src/src/native/unix/configure Mon Nov  4 20:41:27 2013
***************
*** 2637,2642 ****
--- 2637,2646 ----
      CFLAGS="$CFLAGS -DCPU=\\\"arm\\\""
      supported_os="arm"
      HOST_CPU=arm;;
+   aarch64*)
+     CFLAGS="$CFLAGS -DCPU=\\\"aarch64\\\""
+     supported_os="aarch64"
+     HOST_CPU=aarch64;;
    *)
      echo "$as_me:$LINENO: result: failed" >&5
  echo "${ECHO_T}failed" >&6
